### PR TITLE
Add sorting through proposals

### PR DESCRIPTION
**Pull request by foxydevloper**
Adds sorting options for Score, Newest, Oldest

https://user-images.githubusercontent.com/12120644/114500231-9ee48000-9bf5-11eb-98bc-89f27f08bbe2.mp4

### CODE REVIEW

**Comment by foxydevloper** [comment]
whoops  i tried squashing the commits and managed to fail miserably I have no idea how to fix this

File: index.html [hunk 1]
```diff
@@ -150,25 +157,33 @@ <h1 style="text-align: center">Godot proposals viewer</h1>
           proposalsLoaded: false,
           // The GitHub issue label to filter the list of proposals.
           labelFilter: '',
+          labelSort: '',
 
           proposalsFiltered() {
+            var sortMethod
```
Comment (Calinou) [inline index.html]:
```suggestion
            let sortMethod;
```

File: index.html [hunk 2]
```diff
@@ -185,7 +200,26 @@ <h1 style="text-align: center">Godot proposals viewer</h1>
...
+              }
+          },
+
+          sortProposalsOldest(a, b) {
+              if (a.number > b.number) {
+                return 1;
+              } else {
+                return -1;
+              }
+          },
```
Comment (Calinou) [inline index.html]:
```suggestion
          sortProposalsOldest(a, b) {
            if (a.number > b.number) {
              return 1;
            }

            return -1;
          },
```

File: index.html [hunk 3]
```diff
@@ -185,7 +200,26 @@ <h1 style="text-align: center">Godot proposals viewer</h1>
...
-            });
+          },
+
+          sortProposalsNewest(a, b) {
+              if (a.number < b.number) {
+                return 1;
+              } else {
+                return -1;
+              }
+          },
```
Comment (Calinou) [inline index.html]:
```suggestion
          sortProposalsNewest(a, b) {
            if (a.number < b.number) {
              return 1;
            }

            return -1;
          },
```

**Review by Calinou — CHANGES_REQUESTED** [review]

**Review by foxydevloper — COMMENTED** [review]
epic

**Review by Calinou — APPROVED** [review]
Thanks! Congratulations for your first merged pull request :tada:

**Pull request merged**